### PR TITLE
apiserver/client: Fixed capitalisation in meter status message

### PR DESCRIPTION
**Pull request by mattyw**
## Description of change
This branch contains 2 cosmetic improvements for meter status in juju status:

1. juju status was displaying inconsistent capitlisation for model and unit meter status.
2. the header meter status header had 3 components: [Meter,Status,Message]. This was wrong. The output header now has the following 3 components: [Entity, Meter Status, Message]
## QA steps

```
juju bootstrap
juju sla essential
juju status
confirm new header and meter status shows "green" rather than "GREEN"
```
## Documentation changes

Any docs referencing the old format need to be updated

## Bug reference

n/a

### CODE REVIEW

File: cmd/juju/status/output_tabular.go [hunk 1]
```diff
@@ -229,7 +229,7 @@ func FormatTabular(writer io.Writer, forceColor bool, value interface{}) error {
 	}
 
 	if metering {
-		outputHeaders("Meter", "Status", "Message")
+		outputHeaders("Entity", "Meter status", "Message")
```
Comment (frankban) [inline cmd/juju/status/output_tabular.go]: Is Entity always a unit? If so, just Unit?

**Comment by mattyw** [comment]
!!test!!

**Comment by mattyw** [comment]
!!build!!

File: cmd/juju/status/output_tabular.go (hunk 1 above)
Comment (alesstimec) [inline cmd/juju/status/output_tabular.go]: unit or model..

**Comment by mattyw** [comment]
LGTM

**Comment by mattyw** [comment]
$$merge$$

**Comment by mattyw** [comment]
$$merge$$

**Comment by jameinel** [comment]
"Meter Status" seems very large of a header for a field that isn't very
wide normally. Is there a wording that would be shorter?

John
=:->

**Pull request merged**